### PR TITLE
Add page to save and load user data to json file

### PR DESCRIPTION
**Pull request by zakpatterson**
Let the user save their data to a file, and read it back out again from a file.

As with the localstorage object, no real sanity checks are applied yet.

### CODE REVIEW

**Comment by thegrims** [comment]
image
can the settings button color be changed to be the same as the other icons?

**Comment by zakpatterson** [comment]
> image
> 
> can the settings button color be changed to be the same as the other icons?

Ah sure I didn't check it in dark mode.

**Comment by zakpatterson** [comment]
image

File: src/redux/External.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+const Exfiltrate = (props: PropsWithChildren<ButtonProps>): ReactElement => {
+  const store = useStore<TaxesState>()
+
+  const { children, ...rest } = props
+
+  const info = store.getState().information
+
+  const onClick = () =>
+    download(
+      `${info.taxPayer.primaryPerson?.lastName ?? 'return_data'}.json`,
```
Comment (thegrims) [inline src/redux/External.tsx:72]: can the file name contain something like ustaxes_return_data regardless of the last name? I think that would make it more clear for users what the file is used for
```suggestion
      `ustaxes_return_data.json`,
```

**Pull request merged**